### PR TITLE
updated the yarn-audit-known-issues file

### PR DESCRIPTION
**Pull request by leepavlouhmcts**
### JIRA link (if applicable) ###
no jira

### Change description ###
a temporary fix, to ignore the audit errors

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by rayliangatos — COMMENTED** [review]
Something looks like abnormal as the whole yarn.lock is modified, need to check whether the pipeline has passed

File: package.json [hunk 1]
```diff
@@ -219,6 +219,8 @@
     "**/**/socket.io": "^2.4.0",
     "**/**/y18n": "^3.2.2",
     "**/**/ssri": "8.0.1",
-    "**/**/jose": "1.28.1"
+    "**/**/jose": "1.28.1",
+    "**/**/**/ssri": "^8.0.1",
```
Comment (Daniel-Lam) [inline package.json:223]: @leepavlouhmcts - is this correct, i.e. `**/**/**/` (3 of them)?

File: package.json [hunk 2]
```diff
@@ -219,6 +219,8 @@
...
+    "**/**/**/**/xmlhttprequest-ssl": "^1.6.2"
```
Comment (Daniel-Lam) [inline package.json:224]: Same as above - there are 4 `**/`

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@leepavlouhmcts - just two comments to confirm. Also, the `yarn-audit-known-issues` file is empty - is this intentional?

File: package.json (hunk 2 above)
Comment (leepavlouhmcts) [reply package.json]: replaced **/**./**/** to **/**/

File: package.json (hunk 1 above)
Comment (leepavlouhmcts) [reply package.json]: replaced **/**./**/** to **/**/

**Pull request closed without merging**